extras/lbryinc/client.go

```diff
@@ -62,9 +69,19 @@ func (c Client) prepareParams(params map[string]interface{}) (string, error) {
 	return form.Encode(), nil
 }
 
+// SetServerAddress overrides the default internal-apis server address.
+func (c *Client) SetServerAddress(s string) {
+	c.serverAddress = s
+}
+
+// GetServerAddress returns currently defined internal-apis server address.
+func (c Client) GetServerAddress() string {
+	return c.serverAddress
+}
+
 func (c Client) doCall(url string, payload string) ([]byte, error) {
```

Comment:
why are Call and doCall two separate functions? i would either make them one function, or rename doCall to better describe what it does